### ISSUE TITLE
don't get default value

### ISSUE THREAD

**Issue by rodevelop2** [netojose/laravel-bootstrap-4-forms#78]
i want add default value for input, like this in edit form {!!Form::text('name', __('validation.attributes.name') , $user->name)!!} this item works before, but don't work now, please fix this.

**Comment by malyar-m** [comment]
Hi there! 
I noticed a similar issue.
Looks like that this problem caused in method:

FormBuilder.php:line 430 
```
private function getValue()
{
        extract($this->get('name', 'value', 'formData'));
        if ($this->isRadioOrCheckbox()) {
            return $value;
        }

        if ($this->hasOldInput()) {
            return old($name, $value);
        }

        return $formData[$name] ?? null;
}
```
Return value
`return $formData[$name] ?? null;`
should be replaced with 
`return $formData[$name] ?? $value;`

**Comment by netojose** [comment]
Fixed on 3.0.4 version
